modules/RTC/TraceablePeerConnection.js

```diff
@@ -44,15 +70,23 @@ function TraceablePeerConnection(ice_config,
     this.updateLog = [];
     this.stats = {};
     this.statsinterval = null;
-    this.maxstats = 0; // limit to 300 values, i.e. 5 minutes; set to 0 to disable
+    /**
+     * limit to 300 values, i.e. 5 minutes; set to 0 to disable
```

Comment:
This comment is weird. Can this value be changed? If not, should we just remove it?